GNUmakefile

```diff
@@ -27,6 +27,14 @@ VETARGS?=-asmdecl -atomic -bool -buildtags -copylocks -methods \
 # API_PKG sets namespace where the API resources are defined
 API_PKG := github.com/openebs/maya/pkg
 
+# Default arguments for code gen script
+OUTPUT_PKG=github.com/openebs/maya/pkg/client/generated
```

Comment:
Let us write comments that explain these paths. Let us be descriptive for new contributors to understand

Reply:
Done